Automattic/wp-calypso
# Add canAccessWordAds selector

#### Changes proposed in this Pull Request

* Adds a canAccessWordAds selector that's based on feature checks.
* Updates WordAds stats to use the new selector rather than the one that uses a utility function.

#### Testing instructions



* Go to /stats/ads/day/ with a site without WordAds being enabled: <img width="1510" alt="image" src="https://user-images.githubusercontent.com/1398304/167946765-03845254-e590-42a7-aa3b-89834525057b.png">
* Go to /stats/ads/day/ with a site without WordAds being enabled and make sure you see stats: <img width="1510" alt="image" src="https://user-images.githubusercontent.com/1398304/167946888-684bbac1-6226-4a6f-beb2-8abea86cad45.png">


Related to p4TIVU-a66-p2
